datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/common/perf_counters.yaml

```diff
@@ -0,0 +1,123 @@
+- name: metrics
+  description: |
+    This mapping defines which metrics to collect from the performance
+    counters on the `server`. For more information, see:
+    https://docs.microsoft.com/en-us/windows/win32/perfctrs/about-performance-counters
+
+    The top-level keys are the names of the desired performance objects:
+
+      metrics:
+        System:
+          <OPTION_1>: ...
+          <OPTION_2>: ...
+        LogicalDisk:
+          <OPTION_1>: ...
+          <OPTION_2>: ...
+
+    The available performance object options are:
+
+      name (required): This becomes the prefix of all metrics submitted for each counter.
+      counters (required): This is the list of counters to collect.
+      tag_name: This is the name of the tag used for instances. For example, if the tag name for
+                the `LogicalDisk` performance object is `disk`, a possible tag would be `disk:C:`.
```

Comment:
This tag may not appear as the above due to datadog's tag rules https://docs.datadoghq.com/getting_started/tagging/#defining-tags

Reply:
Yup the backend would then normalize this example to `disk:C`

Reply:
addressed!